src/pages/resultsView/mutualExclusivity/MutualExclusivityUtil.tsx

```diff
@@ -41,6 +41,18 @@ export function calculateAdjustedPValue(pValue: number, count: number): number {
     return value > 1 ? 1 : value;
 }
 
+export function calculateQValue(sortedData: MutualExclusivity[]): MutualExclusivity[] {
+    const DataLength = sortedData.length;
+    for (let i = DataLength - 1; i >= 0; i--) {
```

Comment:
for loop is fast but makes code hard to read and error prone.   you should use array.forEach unless there is a demonstrable performance issue.   if you are dealing with a lot of elements (e.g. > 10,000) and you can see rendering performance is suffering, then optimize loops, otherwise, code for readability